android/app/src/main/java/com/microsoft/codepush/common/utils/CodePushLogUtils.java

```diff
@@ -0,0 +1,46 @@
+package com.microsoft.codepush.common.utils;
+
+import android.support.annotation.NonNull;
+
+import com.microsoft.appcenter.crashes.Crashes;
+
+import java.lang.reflect.Method;
+import java.util.Map;
+
+/**
+ * Utils for tracking in-app sdk exceptions.
+ * Represents wrapper on {@link Crashes} methods.
+ */
+public class CodePushLogUtils {
+
+    /**
+     * Represents wrapper on {@link Crashes#trackException(Throwable)} method.
+     *
+     * @param throwable exception instance.
+     * @param shouldLog <code>true</code> if log exception on device.
+     */
+    public static void trackException(Throwable throwable, boolean shouldLog) {
+        try {
+            Method method = Crashes.class.getMethod("trackException", Throwable.class);
+            method.invoke(throwable);
+        } catch (Exception e) {
+            /* Do nothing. */
```

Comment:
Please provide more info in comment why we "Do nothing" while tracking an exception (app secret has not been specified, bad request or etc.)